py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -25,6 +23,8 @@
 )
 from torch_tensorrt.fx.types import TRTTensor
 
+import tensorrt as trt
+
```

Comment:
Why is this change here? Is this a pre-commit change?

Reply:
Precommit